scipy/stats/_discrete_distns.py

```diff
@@ -409,6 +409,17 @@ def _logsf(self, k, M, n, N):
             res.append(logsumexp(self._logpmf(k2, tot, good, draw)))
         return np.asarray(res)
 
+    def _logcdf(self, k, M, n, N):
+        """
+        More precise calculation than log(cdf)
```

Comment:
Could you delete this docstring, as well as the docstrings for `_sf` and `_logsf`? They don't add any information.